CMakeLists.txt

```diff
@@ -50,6 +50,8 @@ option(ENABLE_WARNINGS "Enable warnings" ON)
 option(ENABLE_NATIVE "Enable native CPU build tuning" OFF)
 option(ENABLE_AVX "Enable AVX support" OFF)
 option(ENABLE_OPENMP "Enable OpenMP" ON)
+option(ENABLE_THREADS "Enable threads" ON)
```

Comment:
Is this something we want the user to specify directly, or should we implicitly set it internally based on the build type?

Reply:
Since these options are usually used by developers, I think it is useful to have this option available here.
As it is now, we can make use of threads implicitly with openBLAS.

Reply:
If this is disabled, we still get a working build?

Reply:
Yes, we get it. Do you want me to remove it?

Reply:
Just to expand into this. I tested in Linux both Threads and OpenMP implementations and both yielded a working build.